Objects/frameobject.c

```diff
@@ -304,10 +300,12 @@ mark_stacks(PyCodeObject *code_obj, int len)
                     stacks[i+1] = next_stack;
                     break;
                 case PUSH_EXC_INFO:
-                    next_stack = push_value(next_stack, Except);
-                    next_stack = push_value(next_stack, Except);
-                    next_stack = push_value(next_stack, Except);
-                    stacks[i+1] = next_stack;
+                case POP_EXCEPT:
+                    /* These instructions only appear in exception handlers, which
+                     * skip this switch ever since the move to zero-cost exceptions
+                     * (their stack remains UNINITIALIZED because nothing sets it).
+                     */
+                    Py_UNREACHABLE();
```

Comment:
Maybe this is overly pedantic, but would a `break` here make it clearer that there is no fall-through?

Reply:
+1

This would also help with static code analyzers that do not understand the unreachable macro.

Reply:
Indeed.